src/redux/BookList/BookList.js

```diff
@@ -0,0 +1,23 @@
+// Action Types
+const CREATE = 'react-bookstore-app/books/CREATE';
+const REMOVE = 'react-bookstore-app/books/REMOVE';
+const initialState = {};
+
+// Reducer
+export default function reducer(state = initialState, action) {
+  switch (action.type) {
+    // do reducer stuff
+    case CREATE:
+      return { ...state };
+
+    case REMOVE:
+      return { ...state };
```

Comment:
- [ ] Kindly on the `CREATE` action you have to add to your list, something like ` return [...state, action.newBook ];` to add the new book to the existing ones.
- [ ] Kindly on the `REMOVE` action you have to remove from your list, here you can use the `filter` method to remove the book from your list.